resubmit.py

```diff
@@ -47,41 +45,31 @@ def modifySchema(helper, workflow, user, group, cache, events, firstLumi, backfi
     if backfill is True, modifies RequestString, ProcessingString, AcquisitionEra
     and Campaign to say Backfill, and restarts requestDate.
     """
+    # list of keys to be removed from the new request schema
+    paramBlacklist = ['AcquisitionEra', 'BlockCloseMaxEvents', 'BlockCloseMaxFiles', 'BlockCloseMaxSize',
+                      'BlockCloseMaxWaitTime', 'CouchURL', 'CouchWorkloadDBName', 'CustodialGroup',
+                      'CustodialSubType', 'Dashboard', 'GracePeriod', 'Group', 'HardTimeout', 'InitialPriority',
+                      'inputMode', 'MaxMergeEvents', 'MaxMergeSize', 'MaxRSS', 'MaxVSize', 'MinMergeSize',
+                      'NonCustodialGroup', 'NonCustodialSubType', 'ProcessingString', 'OutputDatasets',
+                      'ReqMgr2Only', 'RequestDate' 'RequestorDN', 'RequestName', 'RequestStatus',
+                      'RequestTransition', 'RequestWorkflow', 'SiteWhitelist', 'SoftTimeout',
+                      'SoftwareVersions', 'SubscriptionPriority', 'Team', 'timeStamp']
```

Comment:
Hi Alan, other than below keys why others has to be removed?
'ReqMgr2Only', 'RequestDate' 'RequestorDN', 'RequestName', 'RequestStatus', 'RequestTransition', 'RequestWorkflow',  'Team', 'timeStamp'